recipes/expat/all/conanfile.py

```diff
@@ -1,5 +1,8 @@
-from conan.tools.microsoft import msvc_runtime_flag
-from conans import ConanFile, CMake, tools
+from conan import ConanFile, tools
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import apply_conandata_patches, collect_libs, copy, patch, rmdir
+from conan.tools.microsoft import msvc_runtime_flag, is_msvc
+from conan.tools.scm import Version
 import os
 
 required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
required_conan_version = ">=1.45.0"
```

due to `conan.tools.microsoft.is_msvc`, maybe even more for other features, I don't know.